packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -825,6 +831,9 @@ class TimeInput extends React.Component {
             />
           </ButtonGroup>
         )}
+        <div id={formatDescriptionId} className={cx('format-text')} aria-label={`Time Format: ${format}`}>
```

Comment:
"Time Format:" should probably be translated.

Reply:
Raised jira for translations.